Backpack/Calendar/Classes/BPKCalendar.m

```diff
@@ -629,11 +628,13 @@ + (UIColor *)fillColorForDateStyle:(BPKCalendarDateCellStyle)style
         case BPKCalendarDateCellStyleNormal:
             return appearance.borderDefaultColor;
         case BPKCalendarDateCellStylePositive:
-            return BPKColor.glencoe;
+            return [BPKColor dynamicColorWithLightVariant:BPKColor.monteverde darkVariant:BPKColor.glencoe];
         case BPKCalendarDateCellStyleNeutral:
             return BPKColor.erfoud;
         case BPKCalendarDateCellStyleNegative:
-            return BPKColor.hillier;
+            return BPKColor.panjin;
+        case BPKCalendarDateCellStyleNoData:
+            return BPKColor.textTertiaryColor;
```

Comment:
I'm unsure about using `textTertiary` here, this is a background, not text so it feels semantically incorrect